pkg/trustedresources/verify_test.go

```diff
@@ -173,26 +168,41 @@ func TestVerifyTask_VerificationPolicy_Success(t *testing.T) {
 	}
 
 	tcs := []struct {
-		name   string
-		task   v1beta1.TaskObject
-		source string
-		signer signature.SignerVerifier
+		name                      string
+		task                      v1beta1.TaskObject
+		source                    string
+		signer                    signature.SignerVerifier
+		verificationNoMatchPolicy string
```

Comment:
does the verificationnomatchpolicy matter for these test cases, since there are matching policies?

Reply:
For this case I think we still need it? I added an unmatched case here

Reply:
I'm still confused-- it looks like all of these tests use the same verification policies. Do these policies match all of the input tasks? If so, the value of verificationNoMatchPolicy doesn't matter. Same comment for TestVerifyPipeline_Success

Reply:
yes they use the same verification policies. But we are passing different `source` to match the policies or not.

Reply:
So there are cases when we don't have any matching policies, but don't fail because the feature flag is set to ignore/warn. We test the same situation from taskref_test.